src/scss/mofo-bootstrap.scss

```diff
@@ -9,18 +10,17 @@
 @import 'custom/colors';
 
 // Core variables and mixins
+@import '../../node_modules/bootstrap/scss/functions';
 @import '../../node_modules/bootstrap/scss/variables';
 @import './overrides/variables';
 @import '../../node_modules/bootstrap/scss/mixins';
-@import './overrides/mixins';
```

Comment:
Where'd this go?

Reply:
Restored and updated to current bootstrap